portal/src/components/roles-and-groups/list/components/RolesAndGroupsBaseList.module.css

```diff
@@ -0,0 +1,14 @@
+.listWrapper {
+  flex: 1 0 0%;
```

Comment:
Use `@apply flex-1` instead of `flex: 1 0 0%`. (Flex shrink 0 should be meaningless if flex basis is 0, because it has nothing to shrink).